### PR TITLE
Create Testing Strategy.md

### PR DESCRIPTION
**Pull request by srmukher**
Create Testing Strategy.md

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/1865fb7022f3e8debece2cd001f4e25e703eb01b'>1865fb7022f3e8debece2cd001f4e25e703eb01b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: packages/react-charting/docs/TestingStrategy.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+3. **wrapTextInsideDonut() -** can be covered by E2E tests as the component is now rendered in a browser.
+
+Sample PRs and Test Plans:
+
+1. Unit test plan and unit tests: [Adding unit tests for donut chart by srmukher · Pull Request #27424 · microsoft/fluentui (github.com)](https://github.com/microsoft/fluentui/pull/27424/files)
+
+Test Plan: [https://microsoftapc-my.sharepoint.com/:w:/g/personal/srmukher_microsoft_com/EZffGTRDJDBBg3nb1CnF8koB3RDDYHxWErXdp7StPqKZeQ?e=72HMsD](https://microsoftapc-my.sharepoint.com/:w:/g/personal/srmukher_microsoft_com/EZffGTRDJDBBg3nb1CnF8koB3RDDYHxWErXdp7StPqKZeQ?e=72HMsD)
+
+1. Component test plan and component tests: [Component testing - Donut chart by srmukher · Pull Request #27033 · microsoft/fluentui (github.com)](https://github.com/microsoft/fluentui/pull/27033)
+
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestingStrategy.md:100]: I believe these test plans have already been added to the docs section in our repo.
Update internal links with public links.

Comment (srmukher) [reply packages/react-charting/docs/TestingStrategy.md]: updated

File: packages/react-charting/docs/TestingStrategy.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
+**Details**
+
+<img width="455" alt="image" src="https://github.com/srmukher/fluentui/assets/120183316/261b1944-b0f3-4e4f-80a3-9f8f88d9f82f">
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestingStrategy.md:3]: the image src is pointing to your local user repo

File: packages/react-charting/docs/TestingStrategy.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+Steps to Measure Test Coverage:
+
+1. Run "yarn test --coverage --coverageReporters=html" from cmd
+2. Open "packages\react-charting\coverage\index.html" to view the test coverage.
+
+Currently, the code coverage for master branch:
+
+Commit Id:
+
+![](RackMultipart20231026-1-6nzx5m_html_e4b63f8ce1082fed.png)
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestingStrategy.md:52]: commit id is not rendering in preview.

File: packages/react-charting/docs/TestingStrategy.md (hunk 2 above)
Comment (srmukher) [reply packages/react-charting/docs/TestingStrategy.md:3]: added as images to images folder.

File: packages/react-charting/docs/TestingStrategy.md (hunk 3 above)
Comment (srmukher) [reply packages/react-charting/docs/TestingStrategy.md]: updated

File: packages/react-charting/docs/TestingStrategy.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
+**Details**
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestingStrategy.md:1]: docs are getting bundled into the release files. Exclude them by adding in .npmignore

Comment (srmukher) [reply packages/react-charting/docs/TestingStrategy.md:1]: added

**Review by AtishayMsft — APPROVED** [review]

**Review by AtishayMsft — APPROVED** [review]

**Pull request merged**